### ISSUE TITLE
Missing dependency

### ISSUE THREAD

**Issue by matusaurio** [Aylur/gnome-extensions#6]
The settings of extension widgets@aylur had an error:
```
Error: Requiring Adw, version none: Typelib file for namespace 'Adw' (any version) not found
```

Stack trace:
```
@/home/eaebnns/.local/share/gnome-shell/extensions/widgets@aylur/prefs.js:3:36
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:204:33
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:13
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:175:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:222:19
start@resource:///org/gnome/gjs/modules/script/package.js:206:5
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```
OS: Ubuntu 20.04.5 LTS x86_64
Gnome version: 3.36.8

**Comment by Mwakamune** [comment]
Same problem here.

OS: Debian GNU/Linux 11 (bullseye) x86_64
DE: GNOME 3.38.6

I installed adwaita-qt and libadwaitaqt-dev, but nothing change.

**Comment by Aylur** [comment]
libadwaita seems to not be available on Ubuntu 20.04. Also I am not even sure that the extension would work on Gnome 3.36, haven't tested it.
